nx.json

```diff
@@ -2,7 +2,6 @@
   "affected": {
     "defaultBase": "main"
   },
-  "npmScope": "ngrx",
```

Comment:
Do we need to rename the `name` in the root `package.json` to `@ngrx/root`, or doesn't that matter (as in, if the prefix is set then we're good to go and the suffix doesn't matter)?

https://nx.dev/reference/nx-json#npm-scope

Reply:
The suffix doesn't matter AFAIK https://github.com/nrwl/nx/blob/master/package.json#L2